objects/sparkle_lovelace/config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
Looks like we have an underscore in the folder name instead of a dash. I think the name should be `sparkle-lovelace`